### ISSUE TITLE
Request for documentation/tutorial

### ISSUE THREAD

**Issue by Boorj** [Pinpickle/tight#5]
Can you make a nice step by step tutorial how to install Bolt, using tight from scratch? Starting with "how to download node.js", continue with "create a folder" ... 
It could be very useful for ~~me~~ people who wants to make a simple installation in a damn simple way :). _(actually i use node, but only to compile *.coffee and *.less files;)_
This tutorial could be probably used in wiki at Bolt.cm, or in future used on Bolt.cm site in "installation ways/options" section.

**Comment by Pinpickle** [comment]
Good idea! It's easy to forget that people who are used to PHP dev could get a bit phased out by this thing called "Node".

I'll work on a "detailed installation" file which I'll link to from the readme.

**Comment by Boorj** [comment]
I can help you if you start that in wiki section, then it could be just transfered to readme =) 
Or you can use google docs or something.

**Comment by Pinpickle** [comment]
Sure thing, [go wild](https://github.com/Pinpickle/tight/wiki/Detailed-Usage)
Thanks for this!
